docs/admin_control_specs.md

```diff
@@ -0,0 +1,51 @@
+# Admin users control over non admin object specs
+
+## Background
+
+Non Admin Controller (NAC) adds the ability to admin users restrict the use of OADP operator for non admin users, by only allowing them to create backup/restores from their namespaces with NonAdminBackup/NonAdminRestore. Admin users may want to further restrict non admin users operations, like forcing a specific NonAdminBackup type. This design enables admin users to restrict which NonAdminBackup/NonAdminRestore specs will be open for non admin users and set custom default values for these specs.
+
+## Goals
+
+- Enable admin users to restrict which NonAdminBackup/NonAdminRestore specs non admin users can use, and set custom default values to these specs
```

Comment:
I was thinking about having 2 options:
- just restrict
- restrict and have custom default value

Thinking about implementation, having just estrict and have custom default value seems easier to implement. Is this ok?